.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] I kindly suggest that you ensure that all your Linters are green when you submit for rereview. 

<img width="628" alt="image" src="https://user-images.githubusercontent.com/52934128/191456658-6e96fd27-c930-448c-a4ee-113259d112f1.png">

Reply:
```
Hello @chigozie1803!

I've just realized that I approved the project instead of requesting for changes.
I am sorry to mislead you. I haven't done that on purpose 😊 
```